src/main/java/org/openstreetmap/atlas/checks/atlas/predicates/ValuePredicates.java

```diff
@@ -1,56 +1,47 @@
 package org.openstreetmap.atlas.checks.atlas.predicates;
 
-import static org.openstreetmap.atlas.checks.constants.CommonConstants.CLOSED_PARENTHESES_CHAR;
-import static org.openstreetmap.atlas.checks.constants.CommonConstants.OPEN_PARENTHESES_CHAR;
-
-import java.util.Stack;
-import java.util.function.Predicate;
+import java.util.ArrayDeque;
+import java.util.Deque;
 
 /**
  * Collection of value based predicates
  *
  * @author brian_l_davis
  */
-public interface ValuePredicates
+public final class ValuePredicates
 {
     /**
      * Tests if the {@link String} contains '(' and ')' that are balanced and closed. Balanced
      * meaning there are the same number of open parentheses as closed. Closed meaning that every
      * open parentheses is followed by a closed parentheses.
+     * 
+     * @param value
+     *            passed as an argument to check if valid parenthesis
+     * @return true if passed argument is valid parenthesis
      */
-    Predicate<String> ARE_PARENTHESES_BALANCED_AND_CLOSED = value ->
+    public static boolean isValidParenthesis(final String value)
```

Comment:
What's the reason for changing from Predicate to boolean ?

Reply:
Good catch. I modified it when changing the interface to class and forgot to keep Predicate back. Will add it.